src/components/Navbar/index.tsx

```diff
@@ -1,6 +1,14 @@
-import React from 'react';
-import './styles.scss';
+import React from "react";
+import "./styles.scss";
+import { Link } from "react-router-dom";
 
 export default function Navbar() {
-  return <nav className="navbar-container">COBDA</nav>;
+  return (
+    <nav className="navbar-container">
+      COBDA
+      <Link to="/sign-up" className="link-container link-container__right">
```

Comment:
```suggestion
      <Link to="/sign-up" className="nav-link">
```
We don't actually need to have two classes for this `<Link/>`.
Any suggestion for class naming? @LilBank

Reply:
Agree, the `flexbox` properties of sending to the right should be in `nav-link`. Furthermore, we should follow **BEM** naming conventions. How about `navbar__item`?

Reply:
At first I thought for the future it will be divided in navbar into two parts. Left side for the logo and right side for other features.